shared/src/state.ts

```diff
@@ -99,6 +100,8 @@ export class ExerciseState {
     public readonly alarmGroups: { readonly [key: UUID]: AlarmGroup } = {};
     @IsIdMap(Client)
     public readonly clients: { readonly [key: UUID]: Client } = {};
+    @IsIdMap(Radiogram)
```

Comment:
This won't work as we always have subtypes of radiograms, i.e. they most likely have 'unexpected' properties. Have a look at `exercise-simulation-activity.ts`, `@IsMultiTypedIdMap` and its use in `simulated-region.ts`